utils/bin/lvim.ps1

```diff
@@ -0,0 +1,8 @@
+$env:HOME = "$HOME"
+$env:USER = "$HOME"
```

Comment:
This will end up with a path that looks like this `$HOME/$HOME/...`.

Reply:
The $HOME and $env:HOME are two different thing. Neovim can't find $HOME if $env:HOME not set